.github/workflows/ci.yml

```diff
@@ -266,3 +266,90 @@ jobs:
         with:
           name: fluvio-k8-logs
           path: /tmp/flv_sc.log
+  k8_cluster_upgrade:
+    name: Kubernetes cluster upgrade test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [infinyon-ubuntu-bionic]
+        rust: [stable]
+    env:
+      FLUVIO_CMD: true
+      FLV_SOCKET_WAIT: 600
+      FLV_CLUSTER_MAX_SC_VERSION_LOOP: 120
+      FLV_CLUSTER_MAX_SC_NETWORK_LOOP: 60
+      FLV_TEST_CONSUMER_WAIT: 300000
+    steps:
+      - uses: actions/checkout@v2
+      - run: helm version
+      - name: Install Rust ${{ matrix.rust }}
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ matrix.rust }}
+          profile: minimal
+          override: true
+      - name: Setup Minikube for Linux
+        if: startsWith(matrix.os, 'infinyon-ubuntu')
+        run: |
+          pkill -f "minikube tunnel" || true
+          minikube delete
+          minikube start --driver=docker --kubernetes-version 1.19.6
+          nohup  minikube tunnel --alsologtostderr > /tmp/tunnel.out 2> /tmp/tunnel.out &
```

Comment:
there is existing script in the `k8-util/minikube.sh`.  Can you create issue to use common that same script to set up everywhere?